src/loki/matrix_action.cc

```diff
@@ -35,7 +35,8 @@ void check_distance(const google::protobuf::RepeatedPtrField<valhalla::Location>
       }
 
       if (path_distance > matrix_max_distance) {
-        throw valhalla_exception_t{154};
+        throw valhalla_exception_t{154, std::to_string(static_cast<size_t>(matrix_max_distance)) +
```

Comment:
is there somewhere a function already that could replace `std::to_string(static_cast<size_t>())`? if not, would it make sense to add one to `midgard.util.h` or so?